lib/main.coffee

```diff
@@ -50,12 +51,13 @@ module.exports =
         parameters.push('--define', 'log_errors=Off')
         text = textEditor.getText()
         return helpers.exec(command, parameters, {stdin: text}).then (output) ->
-          regex = /error:\s+(.*?) on line (\d+)/g
-          messages = []
-          while((match = regex.exec(output)) isnt null)
-            messages.push
-              type: "Error"
-              filePath: filePath
-              range: helpers.rangeFromLineNumber(textEditor, match[2] - 1)
-              text: match[1]
-          return messages
+          return new Promise (resolve) ->
```

Comment:
That was part of my attempt to fix the ".then undefined" errors I was seeing, removed.